crates/ruma-common/src/identifiers/macros.rs

```diff
@@ -90,18 +101,47 @@ macro_rules! owned_identifier {
             }
         }
 
-        #[cfg(feature = "serde")]
```

Comment:
(why was this removed?)

Reply:
Since the crate merge, there is no more serde feature. It only made sense while ruma-identifiers was a separate crate.